DSharpPlus.Rest/DiscordRestClient.cs

```diff
@@ -1216,6 +1216,29 @@ public Task DeleteWebhookAsync(ulong webhook_id, string reason, string webhook_t
         /// <returns></returns>
         public Task<DiscordMessage> ExecuteWebhookAsync(ulong webhook_id, string webhook_token, DiscordWebhookBuilder builder)
             => this.ApiClient.ExecuteWebhookAsync(webhook_id, webhook_token, builder);
+
+        /// <summary>
+        /// Edits a previously-sent webhook message.
+        /// </summary>
+        /// <param name="webhook_id">Webhook id</param>
+        /// <param name="webhook_token">Webhook token</param>
+        /// <param name="messageId">The id of the message to edit.</param>
+        /// <param name="content">The content of the message.</param>
+        /// <param name="embeds">The embeds in the message.</param>
+        /// <param name="mentions">The allowed mentions in the message.</param>
+        /// <returns>The modified <see cref="DiscordMessage"/></returns>
+        public Task<DiscordMessage> EditWebhookMessageAsync(ulong webhook_id, string webhook_token, ulong messageId, Optional<string> content, IEnumerable<DiscordEmbed> embeds = null, IEnumerable<IMention> mentions = null)
```

Comment:
This should be like the ExecuteBuilderAsync method and the MessageCreate/Edit endpoints.  It should take a builder, then pass the builder through

Reply:
I based this on the overloads for message edit which take `Optional` arguments, At any rate, I think a builder would be overkill for such a minor and possibly very rarely used endpoint.

Reply:
Weather or not if it will be used frequently or not should have 0 baring on if it should for consistency sake match how the other parts of the lib are doing it.